docs/02-user-scenario-for-software-migration.md

```diff
@@ -52,40 +52,347 @@ sequenceDiagram
     deactivate Browser
 ```
 
-## User scenarios
+## Register the source computing environment and extract software information
 
 > [!NOTE]
-> 현재 도출된 사용자 시나리오 초안 입니다.
-> 향후, 각 시나리오 별로 Sequence Diagram을 작성할 예정 입니다.
-> 참고 - [Computing Infrastructure Migration Scenario](./01-user-scenario-for-computing-infra-migration.md)
+> ✅ This section is UPDATED.
 
-(소스 컴퓨팅 환경에서 SW 형상 정보(Raw data) 또는 소스 SW 모델(Refined) 추출)
+: Participants: Butterfly, Honeybee, Source computing environment
 
-1. (사용자가) CM portal 에서 소스 그룹 및 연결 정보 등록 요청 :arrow_right: (Honeybee API 호출됨) Honeybee가 대상 Server들에 Agent를 설치함 (비동기)
-2. (사용자가) CM portal 에서 소스 컴퓨팅 환경의 SW 분석 및 추출 요청 :arrow_right: (Honeybee API 호출됨) Honeybee가 SW 형상 정보 / 소스 SW 모델을 제공함
+```mermaid
+sequenceDiagram
+    participant User as User
+    participant WebConsole as Web Console
+    participant Butterfly as Butterfly
+    participant Honeybee as Honeybee
+    participant SrcEnv as Source Computing Environment
+
+    %% Step 0: User accesses Source Computing Environment Management View
+    User->>WebConsole: Access the view of Source Computing Environment Management
+    activate WebConsole
+    WebConsole->>Butterfly: Request the Source Computing Environment Management view
+    activate Butterfly
+    Butterfly-->>WebConsole: Respond with the view
+    deactivate Butterfly
+    WebConsole-->>User: Display Source Environment Management Interface
+    deactivate WebConsole
+
+    %% Step 1: Register Source Group
```

Comment:
I think the term `Source Group` might not be widely understood.
Could you add a short note explaining what it means, or include a link to a document that defines it?